griptape/drivers/prompt/claude_prompt_driver.py

```diff
@@ -0,0 +1,33 @@
+from typing import Optional
+import anthropic
+from attr import define, field, Factory
+from griptape.artifacts import TextArtifact
+from griptape.drivers import BasePromptDriver
+from griptape.tokenizers import TiktokenTokenizer
+
+
+@define
+class ClaudePromptDriver(BasePromptDriver):
+    api_key: str = field(kw_only=True, metadata={"env": "CLAUDE_API_KEY"})
+
+    tokenizer: TiktokenTokenizer = field(
+        default=Factory(lambda self: TiktokenTokenizer(model=self.model), takes_self=True),
+        kw_only=True
+    )
+
+    def try_run(self, value: any) -> TextArtifact:
+        return self.__run_completion(value)
+
+    def __run_completion(self, value: str) -> TextArtifact:
+        client = anthropic.Client(self.api_key)
+
+        response = client.completion(
+            prompt=f"{anthropic.HUMAN_PROMPT}{value}{anthropic.AI_PROMPT}",
```

Comment:
Could you add a brief comment here explaining why the prompt has to be formed this way?